<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>1550</x>
     <y>1020</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1901</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>32</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fragebogen</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkbox_agree">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>950</y>
     <width>881</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string> Ich wurde vom Versuchsleiter über den Ablauf und die Ziele der Studie aufgeklärt.
 Ich bin mit der geplanten Verwendung der erhobenen Daten einverstanden.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hint">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>840</y>
     <width>521</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>selten: weniger als 1x/Woche, unregelmäßig
gelegentlich: weniger als 1x/Tag, mehr als 1x/Woche
häufig: regelmäßig, quasi täglich</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gender">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>1151</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Geschlecht</string>
   </property>
   <widget class="QRadioButton" name="radio_button_gender_no_answer">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Keine Angabe</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_gender_female">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>weiblich</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_gender_diverse">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>divers</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_gender_male">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>128</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>männlich</string>
    </property>
   </widget>
   <zorder>radio_button_gender_no_answer</zorder>
   <zorder>radio_button_gender_female</zorder>
   <zorder>radio_button_gender_diverse</zorder>
   <zorder>radio_button_gender_male</zorder>
   <zorder>gender_2</zorder>
  </widget>
   <widget class="QGroupBox" name="color_blindness">
      <property name="geometry">
         <rect>
            <x>80</x>
            <y>430</y>
            <width>1151</width>
            <height>80</height>
         </rect>
      </property>
      <property name="title">
         <string>Haben Sie eine Farbsehschwäche?</string>
      </property>

      <widget class="QRadioButton" name="color_blindness_yes">
         <property name="geometry">
            <rect>
            <x>120</x>
            <y>40</y>
            <width>128</width>
            <height>31</height>
            </rect>
         </property>
         <property name="text">
            <string>ja</string>
         </property>
      </widget>

      <widget class="QRadioButton" name="color_blindness_no">
         <property name="geometry">
            <rect>
            <x>30</x>
            <y>40</y>
            <width>71</width>
            <height>31</height>
            </rect>
         </property>
         <property name="text">
            <string>nein</string>
         </property>
      </widget>
   </widget>
  <widget class="QGroupBox" name="constraints">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>540</y>
     <width>1151</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Fühlen Sie sich momentan in Ihrer geistigen oder körperlichen Leistungsfähigkeit eingeschränkt? (z.B. durch Krankheit, Alkohol, etc.)</string>
   </property>
   <widget class="QRadioButton" name="radio_button_constraints_yes">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>128</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ja</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_constraints_no">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>nein</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="action_games">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>640</y>
     <width>1151</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Wie oft spielen Sie Actionspiele (z.B. Egoshooter, Action-RPGs, Beat'em ups)?</string>
   </property>
   <widget class="QRadioButton" name="radio_button_action_games_sometimes">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>128</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>gelegentlich</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_action_games_often">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>häufig</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_action_games_rarely">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>selten</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_action_games_never">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>nie</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="strategy_games">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>760</y>
     <width>1151</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Wie oft spielen Sie Echtzeitstrategiespiele/MOBA-Spiele?</string>
   </property>
   <widget class="QRadioButton" name="radio_button_strategy_games_sometimes">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>128</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>gelegentlich</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_strategy_games_never">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>nie</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_strategy_games_rarely">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>selten</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_strategy_games_often">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>häufig</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="age">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>100</y>
     <width>1151</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Alter</string>
   </property>
   <widget class="QComboBox" name="comboBox_age">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gender_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>1151</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Händigkeit</string>
   </property>
   <widget class="QRadioButton" name="radio_button_hand_right">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechtshänder</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_button_hand_left">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>128</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Linkshänder</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
